Sources/SmithyTestUtil/RequestTestUtil/AuthTestUtil/SelectNoAuthScheme.swift

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import ClientRuntime
+import class Smithy.Context
+import struct SmithyHTTPAuthAPI.SelectedAuthScheme
+
+public struct SelectNoAuthScheme: SelectAuthScheme {
+    public static let noAuthScheme = SelectedAuthScheme(
+        schemeID: "smithy.api#noAuth",
```

Comment:
`SelectAuthScheme` that picks the `smithy.api#noAuth` scheme, to be used in tests